### PR TITLE
mitohifi version bump to v3

### PR DESCRIPTION
**Pull request by abueg**
updating docker link to the new mitohifi version + adding some new outputs introduced in v3

### CODE REVIEW

File: tools/mitohifi/macros.xml [hunk 1]
```diff
@@ -11,12 +11,12 @@
     </xml>
     <xml name="requirements">
         <requirements>
-            <container type="docker">quay.io/bgruening/mitohifi:2_galaxy0</container>
+            <container type="docker">ghcr.io/marcelauliano/mitohifi:master</container>
```
Comment (bgruening) [inline tools/mitohifi/macros.xml:14]: @abueg let me know when you are happy with this Container. I will archive it and provide a version for reproducibility.

Happy holidays.

Comment (abueg) [reply tools/mitohifi/macros.xml:14]: OK, i'll check with Marcela to see if the current version of the master is good for archiving, and get back to you 👍🏼 thank you, and happy holidays! 🐰

Comment (abueg) [reply tools/mitohifi/macros.xml:14]: hi @bgruening ! Marcela said the current version of master is good to go for archiving 👍🏼

Comment (bgruening) [reply tools/mitohifi/macros.xml:14]: Ok, I'm pushing it to quay.io atm.

The tests are failing here at the moment.

Comment (bgruening) [reply tools/mitohifi/macros.xml:14]: quay.io/bgruening/mitohifi:3_galaxy0 is now available :)

**Review by bgruening — APPROVED** [review]

**Review by bgruening — APPROVED** [review]

**Pull request merged**